snapm/manager/_systemd.py

```
@@ -0,0 +1,212 @@
+            "unit(%s) state load: %s, active: %s",
+            unit_name,
+            load_state,
+            active_state,
+        )
+
+        if load_state == "loaded":
+            if active_state == "active":
+                return UnitStatus.RUNNING
+            if active_state == "inactive":
+                return UnitStatus.ENABLED
+        return UnitStatus.INVALID  # pragma: no cover
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)